src/mmw/js/src/analyze/controllers.js

```diff
@@ -55,8 +57,11 @@ var AnalyzeController = {
             });
 
         App.state.set('current_page_title', 'Geospatial Analysis');
-
         App.rootView.footerRegion.show(analyzeWindow);
+
+        analyzeWindow.model.get('aoiCensus').done(function(aoiCensus) {
+            App.aoiCensus = aoiCensus;
```

Comment:
`App.currentProject` is possibly undefined at the point that I need to store the area of interest in `analyze/controllers.js`.  I am a little bit nervous about reusing this name to store an incompatible object.